### PR TITLE
Move cassandra add collection to react

### PR DESCRIPTION
**Pull request by hardiknai-techm**
https://user-images.githubusercontent.com/80053762/117774792-76cd5880-b257-11eb-95f4-f0a206d0443f.mp4

### CODE REVIEW

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+    formErrorDetail: "",
+    id: "cassandraaddcollectionpane",
+    isExecuting,
+    title: "Add Table",
+    submitButtonText: "Apply",
+    onClose: closePanel,
+    onSubmit: submit,
+  };
+  return (
+    <GenericRightPaneComponent {...genericPaneProps}>
```
Comment (southpolesteve) [inline src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx:293]: Remove `GenericRightPaneComponent`. I believe we stopped using this in the other panels correct?

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.test.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const props = {
+  explorer: new Explorer(),
+  closePanel: (): void => undefined,
+  cassandraApiClient: new CassandraAPIDataClient(),
+};
+
+describe("CassandraAddCollectionPane  Pane", () => {
+  beforeEach(() => render(<CassandraAddCollectionPaneF {...props} />));
+
+  // it("should render Default properly", () => {
```
Comment (southpolesteve) [inline src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.test.tsx:15]: Remove this or renable it

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.test.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  it("click on is Create new keyspace", () => {
+    fireEvent.click(screen.getByLabelText("Create new keyspace"));
+    expect(screen.getByLabelText("Provision keyspace throughput")).toBeDefined();
+  });
+  it("click on Use existing", () => {
+    fireEvent.click(screen.getByLabelText("Use existing keyspace"));
+  });
+  it("click on Provision keyspace throughput ", () => {
+    fireEvent.click(screen.getByLabelText("Provision keyspace throughput"));
+    // expect(screen.getByLabelText("Table throughput")).toBeDefined();
```
Comment (southpolesteve) [inline src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.test.tsx:28]: Remove this or uncomment it

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx [hunk 4]
```diff
@@ -0,0 +1,452 @@
...
+import * as TelemetryProcessor from "../../../Shared/Telemetry/TelemetryProcessor";
+import { userContext } from "../../../UserContext";
+import * as AutoPilotUtils from "../../../Utils/AutoPilotUtils";
+import { ThroughputInput } from "../../Controls/ThroughputInput/ThroughputInput";
+import Explorer from "../../Explorer";
+import { CassandraAPIDataClient } from "../../Tables/TableDataClient";
+import {
+  GenericRightPaneComponent,
+  GenericRightPaneProps,
+} from "../GenericRightPaneComponent/GenericRightPaneComponent";
```
Comment (southpolesteve) [inline src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx:21]: Said this below, but my understand is we are using RightPaneForm now and getting rid of this componenet

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx (hunk 1 above)
Comment (hardiknai-techm) [reply src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx:293]: done. yes correct

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.test.tsx (hunk 3 above)
Comment (hardiknai-techm) [reply src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.test.tsx:28]: done

File: src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx (hunk 4 above)
Comment (hardiknai-techm) [reply src/Explorer/Panes/CassandraAddCollectionPaneF/CassandraAddCollectionPaneF.tsx:21]: replace with RightPaneForm

**Pull request merged**